src/plugins/ipmi_intf.c

```diff
@@ -128,17 +128,40 @@ struct ipmi_intf * ipmi_intf_table[] = {
 	NULL
 };
 
+/* get_default_interface - return the interface that was chosen by configure
+ *
+ * returns a valid interface pointer
+ */
+static struct ipmi_intf *get_default_interface(void)
+{
+	static const char *default_intf_name = DEFAULT_INTF;
+	struct ipmi_intf ** intf;
+	for (intf = ipmi_intf_table; intf && *intf; intf++) {
+		if (strncmp(default_intf_name, (*intf)->name,
+		            strlen(default_intf_name)) == 0)
```

Comment:
Missed this. What's the purpose of `strncmp` here? Why not just `strcmp` ? I've been lately eradicating these weird comparisons everywhere throughout the code. If you were just mimicking the habits of previous contributors, please don't. And please don't compare `strXcmp` to 0 explicitly. This is idiomatic enough:

`if (!strcmp(default_intf_name, (*intf)->name)) {`

Reply:
It was a cut and paste from below. Fixed.